mailgreen/controller/mail_controller.py

```
@@ -163,3 +182,96 @@ async def get_sender_details(
+    for gmail_msg_id in payload.message_ids:
+        try:
+            # Gmail API 호출: 메일을 휴지통으로 이동
+            service.users().messages().trash(userId="me", id=gmail_msg_id).execute()
+            deleted_ids.append(gmail_msg_id)
+        except HttpError as e:
+            # Gmail API 호출 실패
+            error_detail = None
+            try:
+                error_detail = e.error_details or e.content.decode()
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)